go.mod

```diff
@@ -1,31 +1,196 @@
 module github.com/textileio/go-libp2p-pubsub-rpc
 
-go 1.16
+go 1.17
 
 require (
 	github.com/gopherjs/gopherjs v0.0.0-20190812055157-5d271430af9f // indirect
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/hashicorp/golang-lru v0.5.5-0.20210104140557-80c98217689d // indirect
-	github.com/hsanjuan/ipfs-lite v1.1.23
+	github.com/hsanjuan/ipfs-lite v1.2.1-0.20220309173135-e3246b1d1b2f
```

Comment:
Using the merge commit after [this PR](https://github.com/hsanjuan/ipfs-lite/pull/207) merge.
At some point some new tag will be created and we can "officialize" the dependency version reference to be pretty.